src/config/configuration.ts

```diff
@@ -57,6 +58,26 @@ const configuration = () => {
     }
   });
 
+  let proposalTypesData;
+
+  try {
+    proposalTypesData = fs.readFileSync("proposalTypes.json", "utf8");
+  } catch (error) {
+    // NOTE: If there is no proposalTypes.json file provided we just use the DEFAULT_PROPOSAL_TYPE type
+    console.warn(error);
+  }
+
+  if (!proposalTypesData) {
+    console.warn(
+      `Proposal types configuration file not provided. The "${DEFAULT_PROPOSAL_TYPE}" type will be used`,
+    );
+  }
+
+  const proposalTypes: Record<string, string> = {
+    ...JSON.parse(proposalTypesData || "{}"),
+    DefaultProposal: DEFAULT_PROPOSAL_TYPE,
+  };
+
```

Comment:
I think this can be included in the jsonConfigFileList

```
  const jsonConfigMap: { [key: string]: object[] | boolean } = {};
  const jsonConfigFileList: { [key: string]: string } = {
    loggers: process.env.LOGGERS_CONFIG_FILE || "loggers.json",
  };
  Object.keys(jsonConfigFileList).forEach((key) => {
    const filePath = jsonConfigFileList[key];
    if (fs.existsSync(filePath)) {
      const data = fs.readFileSync(filePath, "utf8");
      try {
        jsonConfigMap[key] = JSON.parse(data);
      } catch (error) {
        console.error(
          "Error json config file parsing " + filePath + " : " + error,
        );
        jsonConfigMap[key] = false;
      }
    }
  });
```

Reply:
Do you want me to hardcode it or add another variable for the `proposalTypes.json` file?